### PR TITLE
Improve auto import message

### PR DESCRIPTION
**Pull request by lnicola**

### CODE REVIEW

**Comment by kjeremy** [comment]
LGTM!

**Comment by SomeoneToIgnore** [comment]
> LGTM!

Did you want to write `bors r+`? :)

File: crates/ra_assists/src/assists/auto_import.rs [hunk 1]
```diff
@@ -64,12 +64,16 @@ pub(crate) fn auto_import<F: ImportsLocator>(
...
 
-    ctx.add_assist_group(AssistId("auto_import"), "auto import", || {
-        proposed_imports
-            .into_iter()
-            .map(|import| import_to_action(import, &position, &path_to_import_syntax))
-            .collect()
-    })
+    ctx.add_assist_group(
+        AssistId("auto_import"),
+        format!("Import {}", path_to_import.syntax()),
```
Comment (SomeoneToIgnore) [inline crates/ra_assists/src/assists/auto_import.rs]: > path_to_import.syntax()

NIT: I think you could have used the `path_to_import_syntax` variable here.

Comment (lnicola) [reply crates/ra_assists/src/assists/auto_import.rs]: Yeah, I just noticed that. Thanks.

**Comment by flodiebold** [comment]
bors r+

:wink:

**Comment by lnicola** [comment]
Looks like the MacOS worker might be stuck: https://github.com/rust-analyzer/rust-analyzer/runs/420083660.

**Comment by kiljacken** [comment]
bors try

**Comment by bjorn3** [comment]
Why are there 3 try responses?

**Comment by lnicola** [comment]
GitHub actions timed out last night and now bors is confused.

**Pull request merged**